cmd/info.go

```diff
@@ -35,7 +36,15 @@ the current directory or from the directory specified with --path.
 		PreRunE:           bindEnv("output", "path"),
 	}
 
+	// Config
+	cfg, err := config.NewDefault()
+	if err != nil {
+		fmt.Fprintf(cmd.OutOrStdout(), "error loading config at '%v'. %v\n", config.File(), err)
+	}
+
+	// Flags
 	cmd.Flags().StringP("output", "o", "human", "Output format (human|plain|json|xml|yaml|url) (Env: $FUNC_OUTPUT)")
+	cmd.Flags().StringP("namespace", "n", cfg.Namespace, "The namespace in which to look for the named function. (Env: $FUNC_NAMESPACE)")
```

Comment:
Why is there not similar logic as in `cmd/deploy.go` - here and in the other commands (`delete` and `list`)?

Reply:
This line was incorrectly included in this PR, thanks for noticing.  I separated out the namespace usage (and tests) each into their own separate PR for easier review.